lib/quiet_quality/tools/brakeman.rb

```diff
@@ -3,6 +3,7 @@
 module QuietQuality
   module Tools
     module Brakeman
+      TOOL_NAME = :brakeman
```

Comment:
I'm not sure I like this pattern - constants don't inherit/override the way methods do, which leads to issues like you appear to have noticed with standardrb's runner needing to reimplement the same same `tool_name` method.

The win is that it does let us define this token in exactly one place each though. So I'd rather either (a) just use the raw symbol in the parsers and runners (and drop the constants) or (b) update the remaining locations that use the raw symbols to not do so. I favor the former, but I'll leave the choice to you :-)

Reply:
I looked for other instances of the symbols and couldn't find them outside of specs - do you think I should use constants there too, if I stick with this approach?

Reply:
The main place I was thinking of was the mapping in the Tools module itself - that's where we look tools up based on their key. The reason you couldn't find them elsewhere was that they're being constructed from strings passed in from config files and cli arguments before being extracted from there, and in the mapping they are in colon-last format, since it's a Hash literal. I may revisit this later, but it's not hurting anything like this.

Reply:
Oooh of course. My mistake!